src/main.rs

```diff
@@ -138,8 +138,8 @@ fn main() -> Result<(), Box<dyn Error>> {
         None                   => DnsSettings::load_default(),
       };
       let settings = settings.expect("Error reading settings!");
+      log_info!("Settings: {:?}", settings);
```

Comment:
You can turn on debug logging with the --features="log_debug". I have an improvement in my current PR that lets you specify the minimum log level you want to see. I removed this from info as a way to reduce console output noise in preparation for a different/more configurable logging output

Reply:
My bad, this was a leftover line from a merge I messed up. Removed in 3154f036f3ad74185b3686db8c8ee358af479c9d.